test/select.jl

```diff
@@ -2707,4 +2707,57 @@ end
     @test_throws ArgumentError select(gdf, :a => [:x, :y, :z])
 end
 
+@testset "additional test of handling of operation specfication in select!/transform!" begin
```

Comment:
```suggestion
@testset "additional test of handling of operation specification in select!/transform!" begin
```
(It's a bit long BTW.)

Reply:
OK - I made it shorter